### PR TITLE
added style, complete the js requirements and update readme

### PR DESCRIPTION
**Pull request by TamaraDimov**
-Create an index.html file with an empty To Do List placeholder <ul>
-Create an index.js file and set an array of some simple to-do tasks (array of objects).
-Create a style.css and set rules for the To Do List
-using webpack to generate a webpack dev server from /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,7 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
+
```
Comment (brytebee) [inline README.md:30]: - [ ] **Optional:** This is my personal advice, you are not required to do it! However, I noticed that sections such as `Contributing`, `Acknowledgement`, `FAQ` in your Readme are without professional content. I suggest that you take a couple of minutes of your free time to customize them as they do not change much from one project to the other. I hope this is useful. 😃

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+    description: 'Wash the dishes',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Buy groceries',
+    completed: false,
+    index: 3,
+  },
+];
```
Comment (brytebee) [inline src/index.js]: - [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index)`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @TamaraDimov👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @TamaraDimov,

Good job working on the changes asked by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used Webpack for project setup 👍🏽
✅ Set up todo list structure as expected 👍🏽 

## Recommendation:

- [ ] We're asked to professionally document all our projects professionally in the README because, Microverse is preparing us to become top-class professional developers hence, our README for each project is advised to have a detailed description in all sections, proper description of the project, it features, how to clone and use the project, how to install all packages, run the app, give acknowledgment and credits to Microverse plus any other supports gained from any resource while building the project and more... This will help set you up a good stance should a recruiter end up looking at one of your projects and help increase your chances of landing a dream job. Kindly consider enhancing your README for all projects, please 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**